etc/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2020 VMware, Inc. All Rights Reserved.
+# Copyright (C) 2020-2022 VMware, Inc. All Rights Reserved.
```

Comment:
You should only change the year when you made changes, or when this was already changed this year but we forgot to change the year. This could probably be automatized.

Reply:
Yeah. I will write a script for this when I get time. I have seen other projects running it once in a year. It's okay if we update our copyright headers.